sdcm/cluster_k8s/__init__.py

```diff
@@ -2741,7 +2743,9 @@ def decommission(self, node: BaseScyllaPodContainer, timeout: int | float = None
         scylla_shards = node.scylla_shards
 
         timeout = timeout or (node.pod_terminate_timeout * 60)
-        with adaptive_timeout(operation=Operations.DECOMMISSION, node=node):
+        with adaptive_timeout(operation=Operations.DECOMMISSION, node=node), DbEventsFilter(
+                db_event=DatabaseLogEvent.RUNTIME_ERROR,
+                line="std::runtime_error.*decommission"):
```

Comment:
May create line a bit specific for line mention in PR description?
it seems, that we can lost real errors which may happened during decommission?